src/components/Reports/ReportsPageRanking/ReportsPageRanking.test.js

```diff
@@ -95,7 +98,7 @@ describe('Reports pages ranking', () => {
   it('should show empty message when dont have pages', async () => {
     const datahubClientDouble = {
       getPagesRankingByPeriod: async () => {
-        return { success: true, value: [] };
+        return { success: true, value: { hasMorePages: false, pages: [] } };
```

Comment:
You have a scenario for `hasMorePages` in `true` right?

Reply:
yes i added a test